package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "build": "turbo run build",
     "build:watch": "turbo run build --parallel -- --watch",
-    "ci:test": "./scripts/ci-test.sh",
+    "ci:test": "pnpm build && ./scripts/ci-test.sh",
```

Comment:
We don't need it.

The CI runs the `build` step before the `test`one.

Reply:
are you sure @arboleya ? 

I added because the coverage-test action wasn't running it on master (when it runs for the second time to compare with current tests)
and tests were broken because of differences in typing

maybe a different approach would be changin the script here instead:
https://github.com/FuelLabs/fuels-ts/blob/master/.github/workflows/test.yaml#L55

Reply:
@arboleya so what happens is:

https://github.com/FuelLabs/fuels-ts/blob/master/.github/workflows/test.yaml#L55

this ArtiomTr/jest-coverage-report-action@v2 action does:
- checkout current PR
- pnpm install
- pnpm ci:test (at this stage is all good, because CI had run pnpm build before)
- checkout master (for comparison)
- pnpm install
- pnpm ci:test (here it breaks, because build was run to PR branch, instead of master)


that's why I include `pnpm build` in `ci:test`. you have other idea on how to fix this ?

I undone the change so you can see the error happening in CI test step

Reply:
> I undone the change so you can see the error happening in CI test step

after I did this, error stopped happening. but I swear we had it 😅🫠

can it be some intermittent thing? 

anyway it's removed I will resolve this conversation, good catch @Dhaiwat10

Reply:
hmmmm I was exploring a bit more and here's the conclusion:

- https://github.com/FuelLabs/fuels-ts/actions/runs/4662084408/jobs/8252067068
  - this run was broken, which bring to my attention the tests broken when comparing to `master` branch. but turns out the problem was _Error: Jest coverage report action failed_ . maybe some network/intermittent problem with the action
- https://github.com/FuelLabs/fuels-ts/actions/runs/4684265545/jobs/8300231734
  - then this one worked, but if you scroll into it, you will see tests are still broken when comparing to master. it's only not breaking CI, but broken tests are there

  
I guess it's something that can be handled in other issue as it's orthogonal to this PR objective/changes

Reply:
The _first_ broken run you mentioned comes from this [flaky](https://github.com/FuelLabs/fuels-ts/actions/runs/4662084408/jobs/8252067068#step:10:1203) test.

The _second_ one is exactly what you described — _good catch_!

However, in #827 we no longer run tests twice; this problem will disappear.

Reply:
I've extracted this fix into a new [PR](https://github.com/FuelLabs/fuels-ts/pull/900).